technical-vision.md

```diff
@@ -1,9 +1,34 @@
 # OpenSSF Technical Vision
 
-We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively, by default, and as a matter of course:
-- Developers can easily learn secure development practices and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
-- Developers, auditors, and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
-- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
-- Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
+An aspirational view of the future we seek to create in 2-5 years:
 
-The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+## The OpenSSF is a trusted partner to affiliated open source foundations and projects, and provides valuable services to those projects and foundations.
+- The OpenSSF is well positioned within the broader ecosystem and has a clearly articulated charter which supports the activities of the broader community of open source practitioners
+- To maintain that trusted relationship, the OpenSSF's mission and value proposition is periodically reviewed with community representatives and constituents, as well as its funding Members
+- OpenSSF resources are responsibly managed and utilized towards this Vision in clear, consistent, transparent, and democratic ways.
+- Where overlap exists between the OpenSSF's mission and that of another organization, the OpenSSF strives to support other organizations through partnership first, and does not engage in "picking winners and losers".
+
+### OpenSSF is a well-respected strategic partner to large OSS foundations and individual OSS projects
+- OpenSSF is clearly positioned within broader ecosystem with a clear, altruistic charter
+- Periodically review the value proposition of OpenSSF with key stakeholders
+  - Joint OKRs on shared execution, utilization of OpenSSF-managed tools & content
```

Comment:
Joint OKRs with who? I'm assuming this is referring to the "large OSS foundations" in the title and not the "individual OSS projects".

Reply:
yes, it was written under the assumption that the larger OSS foundations might desire and explicitly agree to having shared OKRs. 

If an individual project maintainer for a project deemed as critical wanted to have a shared roadmap with the OpenSSF to drive improvement in their security posture in a way that needed high-touch engagement from the foundation, I would think the foundation would be supportive of that.